apps/console/src/features/users/components/user-groups-edit.tsx

```diff
@@ -95,6 +95,7 @@ export const UserGroupsList: FunctionComponent<UserGroupsPropsInterface> = (
     const [ assignedGroups, setAssignedGroups ] = useState<RolesMemberInterface[]>([]);
     const [ isPrimaryGroupsLoading, setPrimaryGroupsLoading ] = useState<boolean>(false);
     const [ isSubmitting, setIsSubmitting ] = useState<boolean>(false);
+    const [ oldGroupList, setOldGroupList ] = useState([]);
```

Comment:
Shall we rename this to `existingGroupList`. WDYT?
Also add the type.

```suggestion
    const [ oldGroupList, setOldGroupList ] = useState<WHICH[]>([]);
```

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3004/commits/28231c0ab636b337b78aee8ccc62359d8a46e486